metrics_proto.go

```diff
@@ -112,13 +73,21 @@ func (se *statsExporter) ExportMetricsProtoSync(ctx context.Context, node *commo
 		if metric.GetMetricDescriptor().GetType() == metricspb.MetricDescriptor_SUMMARY {
 			summaryMtcs := se.convertSummaryMetrics(metric)
 			for _, summaryMtc := range summaryMtcs {
+				if err := se.createMetricDescriptor(ctx, summaryMtc, se.defaultLabels); err != nil {
```

Comment:
Calling create metric descriptor for every request will be expensive. I think we can just omit this step and let Stackdriver auto creates it: https://cloud.google.com/monitoring/custom-metrics/creating-metrics#auto-creation.

Reply:
Also for built-in metrics we cannot re-create the descriptors.

Reply:
createMetricDescriptor will look for:
1) If we already did this
2) If buildin mark as already done this and return
3) Do a create if successful then mark as already done.